libs/content/workspace/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod theme;
 pub mod widgets;
 pub mod workspace;
 
+pub use lb_rs;
```

Comment:
I'm fairly certain that this means the compiler is free to optimize lb_rs for workspace (and ultimately workspace_ffi) differently. It may not, but it may. Could you decouple this change from your PR for now?

You can feel free to continue doing this locally for now, and at release time it won't matter as both libraries will be built.

Your finding that the functions from core will be included twice is a good discovery, it'll simplify the design of workspace_ffi providing core once. This means the only additional thing build_swift_libs has to do is import the header file. That'll mean you won't ever have to build both again and we won't have to include all the static objects for core twice. It'll be a good cleanup in a few regards when we do it properly.

Reply:
Sure, i'll change it back.